src/client/pages/ToeristischeVerhuurDetail/VergunningVerhuur.tsx

```diff
@@ -1,11 +1,97 @@
+import { useMemo } from 'react';
 import {
   ToeristischeVerhuurBBVergunning,
   ToeristischeVerhuurVergunningaanvraag,
 } from '../../../server/services';
 import { defaultDateFormat } from '../../../universal/helpers';
+import { PageContent } from '../../components';
 import InfoDetail, {
   InfoDetailGroup,
 } from '../../components/InfoDetail/InfoDetail';
+import StatusLine, {
+  StatusLineItem,
+} from '../../components/StatusLine/StatusLine';
+import styles from './ToeristischeVerhuurDetail.module.scss';
+import { DocumentDetails } from '../VergunningDetail/DocumentDetails';
+
+function getStatusBB(decision: string) {
+  if (decision.includes('Verleend')) {
```

Comment:
ben je er niet zeker van dat de decision daadwerkelijk 'Verleend' 'Geweigerd' of 'Ingetrokken' is?

Reply:
Nouja als we het RD moeten geloven? Anders weet ik niet hoe ik deze logica moet verwerken

Reply:
Oooh, ik zie 't. Je kunt ook iets doen als: `['Verleend','Ingetrokken','Geweigerd'].some(status => decision.includes(status))`

Reply:
Mja, laatmaar zo.